src/ingest_data.py

```diff
@@ -0,0 +1,179 @@
+import argparse
+import logging
+import os
+import pickle
+
+import numpy as np
+import pandas as pd
+from sklearn.impute import SimpleImputer
+from sklearn.model_selection import StratifiedShuffleSplit, train_test_split
+
+from modules import income_cat_proportions, load_housing_data
+
+housing = load_housing_data()
+
+parser = argparse.ArgumentParser()

```

Comment:
This section should always be in the block `if __name__ == "__main__"`, if you want to read more, follow the link below.
[Why use if __name__ == "__main__"](https://realpython.com/if-name-main-python/#:~:text=Syntactically%2C%20Python's%20if%20__name,line%201%20evaluates%20to%20True%20.)

Reply:
I'm talking about adding the arguments to the script.